manifests/autotune/performance-profiles/resource_optimization_local_monitoring.yaml

```diff
@@ -23,17 +23,17 @@ slo:
 
     aggregation_functions:
     - function: avg
-      query: 'avg by(container, namespace) (kube_pod_container_resource_requests{container!="", container!="POD", pod!="", resource="cpu", unit="core", namespace="$NAMESPACE$", container="$CONTAINER_NAME$"})'
+      query: 'avg by(container, pod, namespace) ((kube_pod_container_resource_requests{container!="", container!="POD", pod!="", resource="cpu", unit="core", namespace="$NAMESPACE$", container="$CONTAINER_NAME$"}) * on(pod, namespace) group_left max by (container, pod, namespace) (kube_pod_status_phase{phase="Running"}))'
```

Comment:
Irrespective of pod_status_phase - we need to gather all the data. We can remove the grouping ` * on(pod, namespace) group_left max by (container, pod, namespace) (kube_pod_status_phase{phase="Running"}))`

Reply:
Updated the queries to remove grouping based on the pod status.